etc/tendrl/tendrl.auth.conf.sample

```diff
@@ -0,0 +1,35 @@
+[general]
+etcd_server = 10.70.43.26
```

Comment:
Why using actuals? Put some dummy. may be 127.0.0.1
Also etcd detail are available as part of common section adn we should use from there. Why declare again in this section?